src/main/java/com/onelogin/sdk/conn/Client.java

```diff
@@ -2438,6 +2442,70 @@ public SAMLEndpointResponse getSAMLAssertionVerifying(String appId, String devid
 		return getSAMLAssertionVerifying(appId, devideId, stateToken, null, null);
 	}
 
+	///////////////////////
+	//  SMART MFA        //
+	///////////////////////
+
+	/**
+	 * Validates a User
+     *
+	 * @param validateUserParams Smart MFA User data (user_identifier, email, phone, context, risk_threshold,
+     *                                                 firstname, lastname, expires_in)
+	 *
+	 * @return SmartMFA
+	 *
+	 * @throws OAuthSystemException - if there is a IOException reading parameters of the httpURLConnection
+	 * @throws OAuthProblemException - if there are errors validating the OneloginOAuthJSONResourceResponse and throwOAuthProblemException is enabled
+	 * @throws URISyntaxException - if there is an error when generating the target URL at the URIBuilder constructor
+	 * @throws ErrorResourceInitialization
+	 *
+	 * @see com.onelogin.sdk.model.SmartMFA
+	 * @see <a target="_blank" href="https://developers.onelogin.com/api-docs/2/smart-mfa/validate-user">Validate a Use documentation</a>
+	 */
+	public SmartMFA validateUser(Map<String, Object> validateUserParams) throws OAuthSystemException, OAuthProblemException, URISyntaxException, ErrorResourceInitialization {
+			int versionId = settings.getVersionId("SMART_MFA_VALIDATE_USER");
+			URIBuilder url = new URIBuilder(settings.getURL(Constants.SMART_MFA_VALIDATE_USER, versionId));
+
+			return (SmartMFA) createResource(SmartMFA.class, validateUserParams, url, versionId);
```

Comment:
This fails, as createResource for API version 2 only checks for 201 response code. Is it ok to check for a 200 response code here also?

Reply:
Yes, let's check for 201 and 200

Reply:
Cool, I have done that.